app/templates/events/view.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if (or (eq session.currentRouteName 'events.view.index') (eq session.currentRouteName 'events.view.export'))}}
-  {{#if (eq model.state 'draft')}}
+  {{#if (and (eq model.state 'draft') (unverified-user-message))}}
```

Comment:
Can you please provide a GIF showing this change?

Reply:
> Can you please provide a GIF showing this change?

![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/56407566/68412420-2ed8fb80-01b2-11ea-8b0e-d57e6b96f15f.gif)
it's working fine .